hack/verify-import-boss.sh

```diff
@@ -30,7 +30,7 @@ kube::golang::setup_env
 
 make -C "${KUBE_ROOT}" WHAT=vendor/k8s.io/code-generator/cmd/import-boss
 
-packages=("k8s.io/kubernetes/pkg/..." "k8s.io/kubernetes/cmd/..." "k8s.io/kubernetes/plugin/..." "k8s.io/kubernetes/test/e2e/framework/...")
+packages=("k8s.io/kubernetes/pkg/..." "k8s.io/kubernetes/cmd/..." "k8s.io/kubernetes/plugin/..." "k8s.io/kubernetes/test/e2e/framework/..." "k8s.io/kubernetes/test/integration/...")
```

Comment:
can you put these on newlines and alphasort for ease of diff going forward?
```suggestion
packages=(
  "k8s.io/kubernetes/cmd/..."
  "k8s.io/kubernetes/pkg/..."
  # etc..
)
```

Reply:
@spiffxp Thanks for your review.
That is a good idea to maintain many places.